src/sections/Meshery/Meshery-integrations/IntegrationsGrid.js

```diff
@@ -139,7 +151,7 @@ const IntegrationsGrid = ({ category, theme, count }) => {
               className={item.isSelected ? "items selected" : "items"}
               onClick={setFilter}
             >
-              {item.name}
+              {item.name} ({item.count})
```

Comment:
@tinniaru3005 due to the new code changes the categories are now not clickable . If you could revert 1 step to the same scheme of code , things will be good .